exchanges/gateio/gateio_test.go

```diff
@@ -1842,16 +1842,16 @@ func TestGetOptionsSpecifiedSettlementHistory(t *testing.T) {
 	if len(pairs) == 0 {
 		t.Skip(err)
 	}
-	pair, err := currency.NewPairFromString(pairs[1])
+	pair, err := currency.NewPairFromString(pairs[0])
 	if err != nil {
 		t.Skip(err)
 	}
 	underlying, err := g.GetUnderlyingFromCurrencyPair(pair)
 	if err != nil {
 		t.Skip(err)
 	}
-	if _, err := g.GetOptionsSpecifiedSettlementHistory(context.Background(), pairs[1], underlying, 0); err != nil {
-		t.Errorf("%s GetOptionsSpecifiedSettlementHistory() error %s", g.Name, err)
+	if _, err := g.GetOptionsSpecifiedContractsSettlement(context.Background(), pairs[0], underlying, 12); err != nil {
```

Comment:
This is still not working. See [here](https://github.com/thrasher-corp/gocryptotrader/pull/1058#discussion_r1025832095)

```
GateIO GetOptionsSpecifiedContractsSettlement() error GateIO unsuccessful HTTP status code: 404 raw response: {"label":"NOT_FOUND"}
```

Reply:
This endpoint has confusing issues.
I think it is better to ask in their API documentation support